src/pages/proposals/test-proposals.json

```diff
@@ -0,0 +1,164 @@
+[
```

Comment:
Can you add a task to remove this testing code?

Reply:
I am planning to remove the test code before the merge but yeah, I have added the [task](https://github.com/api3dao/api3-dao-dashboard/issues/506) in case I forget